tests/integration_tests/example_workspaces/pkl_cache/.bazelversion

```diff
@@ -1 +1 @@
-7.1.2
+7.2.1
```

Comment:
For what it's worth I _believe_ we can make these a symlink to the root .bazelversion to avoid having to manually keep them in sync. Haven't tested it in this project though.